### PR TITLE
Millitary slang cheat sheet: Added more terms

### PR DESCRIPTION
**Pull request by mayank7**
Added more terms to Millitary slang cheat sheet

IA Page: https://duck.co/ia/view/millitary_slang

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by mayank7** [comment]
Reference: http://www.military.com/join-armed-forces/military-terms-and-jargon.html

File: share/goodie/cheat_sheets/json/military-slang.json [hunk 1]
```diff
@@ -191,6 +207,22 @@
...
+      {
+        "key": "Hardball",
+        "val": "A hard-surfaced road"
+      },
+      {
+        "key": "Hardened Site",
+        "val": "A structure usually built under rock or concrete designed to withstand conventional, nuclear, biological, and chemical attack"
+      },
+      {
+        "key": "Helo ",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/military-slang.json]: Nit: trailing space

Comment (moollaza) [inline share/goodie/cheat_sheets/json/military-slang.json]: Please remove that ;)

**Comment by moollaza** [comment]
Thanks! This LGTM :+1:

**Pull request merged**